nephio/optional/rootsync/rootsync.yaml

```diff
@@ -11,3 +11,4 @@ spec:
     auth: token
     secretRef:
       name: example-cluster-name-access-token-configsync
+    period: 15s
```

Comment:
@efiacor can you add in the PR the description of this parameter period? This way we can keep track of having this value as 15.

Reply:
done